### ISSUE TITLE
Datasets on Play Stats

### ISSUE THREAD

**Issue by adataylor** [ajanata/PretendYoureXyzzy#152]
Looking at the data stored, it seems that the only data stored by you guys is the cards and their groupings, not any info on how play tends to go (ex, which pairings win, which lose, etc).

A) Is there any existing data I could use for a research project?
B) If not, is there any chance I could add a logging line to (anonymously) track round outcomes to my own DB?

**Comment by ajanata** [comment]
Correct, there is currently not any logging for this. It will be coming in the next few months, though, as I have a very good reason to do so now (I do not feel at liberty to go into details right now). I expect most of the data will be available publicly.
